samples/cors/ts/README.md

```diff
@@ -0,0 +1,121 @@
+# Using Azure Key Vault in a Web Application
+
+Browser security prevents a web page from making requests to a different domain than the one that served the web page. This restriction is called the same-origin policy. The same-origin policy prevents a malicious site from reading sensitive data from another site. Sometimes, you might want to allow other sites to make cross-origin requests to your app. That's where Cross Origin Resource Sharing (CORS) policies become necessary.
+
```

Comment:
A link here to more CORS info could be useful. My mind went  to the CORS specification [link](https://fetch.spec.whatwg.org/), but it could be something else, like in the Mozilla docs, or in the Microsoft docs, of course. Up to you.